src/com/mediaengine/rpc/rpc_cdm_mediaengine_handler.cc

```diff
@@ -19,6 +19,23 @@
 #include <cdm_logging.h>
 #include <opencdm_xdr.h>
 
+#ifdef OCDM_SDP_PROTOTYPE
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/socket.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include "linux/ion.h"
+
+// TODO Organize a compile time flag
+// i.MX8M VPU secure heap
+#define ION_SECURE_HEAP_ID_DECODER 2
```

Comment:
Would be good to do this "todo" so we can have it working on both hikey and iMX8M